feat: add new doc related page view events (#752)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sdks/marketing-analytics-browser/index.md

--8<-- "includes/sdk-ts-browser/marketing-analytics.md"

> The following are the event info got tracked in the page view events

Reviewer: ```suggestion
The following information is tracked in the page view events.
```
